src/cache.ts

```diff
@@ -32,9 +32,24 @@ const supportedPackageManager: PackageManager[] = [
     path: [join(os.homedir(), '.gradle', 'caches'), join(os.homedir(), '.gradle', 'wrapper')],
     // https://github.com/actions/cache/blob/0638051e9af2c23d10bb70fa9beffcad6cff9ce3/examples.md#java---gradle
     pattern: ['**/*.gradle*', '**/gradle-wrapper.properties']
+  },
+  {
+    id: 'sbt',
+    path: [
+      join(os.homedir(), '.ivy2', 'cache'),
+      join(os.homedir(), '.sbt'),
+      getCoursierCachePath()
```

Comment:
Hello @fmeriaux. Do we need to add Coursier ? Does it required for sbt ?

Reply:
According to https://get-coursier.io/docs/sbt-coursier

> sbt 1.3.x ships with coursier support out-of-the-box, and that support is enabled by default. You shouldn't need to add sbt-coursier to sbt 1.3.x builds in most cases.

Reply:
@dmitry-shibanov is it ok for you?